cfpb/consumerfinance.gov
# RAF tool: update data

Update compiled JS for RAF tool containing latest JSON with program URL changes.

## Changes

- Update RAF JS


## How to test this PR

1.  Check out branch
2. Run gulp copy
3. Create browse page and add RAF tool block
4. Verify that tool works as expected
5. Verify that program URL updates are reflected:
- Gilbert, Arizona: https://www.gilbertaz.gov/residents/community-and-neighborhood-services/community-resources/foreclosure-prevention-predatory-lending-information
- Oakland, California: https://www.oaklandca.gov/resources/housing-resources-erap-emergency-rental-assistance
- Riverside, California: https://www.riversideca.gov/homelesssolutions/housing-authority/riverside-rental-assistance-program
- Colorado: https://cdola.colorado.gov/rental-mortgage-assistance
- Collier County, Florida: https://www.colliercountyhousing.com/community-assistance-program/
- New Hampshire: https://www.nhhfa.org/emergency-rental-assistance
- Northampton County, Pennsylvania: https://www.northamptoncounty.org/CMTYECDV/Pages/default.aspx
- Philadelphia, Pennsylvania: https://phlrentassist.org/
- Spartanburg County, South Carolina: https://www.shasc.org/erap
- Tarrant County, Texas: https://www.tarrantcounty.com/en/county/social-services/rental-assistance-application.html
